src/recipes.jl

```diff
@@ -325,6 +319,22 @@ function Makie.plot!(gp::GraphPlot)
         visible = arrow_show_m,
         gp.arrow_attr...)
 
+
+    # update edge paths to line up with arrow heads if arrow_shift = :end
+    edge_paths = gp[:edge_paths] = @lift map($init_edge_paths, $arrow_pos, eachindex($arrow_pos)) do ep, ap, i
```

Comment:
i think `@lift map` is redundent as observables.jl overloads map directly https://juliagizmos.github.io/Observables.jl/stable/#Base.map-Union{Tuple{F},%20Tuple{F,%20AbstractObservable,%20Vararg{Any}}}%20where%20F